test/unit/file.test.ts

```diff
@@ -1,3 +1,4 @@
+import { Readable, PassThrough } from 'stream';
```

Comment:
NP / this import should be below the header (and `Readable` import can be dropped)